Migrate crictl to docker-cli mapping to references (#31316)
kubernetes/website: content/en/docs/reference/tools/map-crictl-dockercli.md

---
title: Mapping from dockercli to crictl
content_type: reference
---

> 

Reviewer: We might want to use the `{{% thirdparty-content %}}` shortcode here (I would).

{{<caution>}}
This page is deprecated and will be removed in Kubernetes 1.27.

> {{</caution>}}

Reviewer: Use a note callout here. Caution is for advice that you should heed if you don't want to put your cluster at risk (warning is for “never do this: _x_” notices).

`crictl` is a command-line interface for {{<glossary_tooltip term_id="cri" text="CRI">}}-compatible container runtimes.
You can use it to inspect and debug container runtimes and applications on a
Kubernetes node. `crictl` and its source are hosted in the
[cri-tools](https://github.com/kubernetes-sigs/cri-tools) repository.

This page provides a reference for mapping common docker cli commands to

> equivalent crictl commands.

Reviewer: nit:
Most times in this page, you should either write “Docker” or `docker`.

It's OK to skip that as this is moving existing content.

> ## Mapping from docker cli to crictl

Reviewer: ```suggestion
## Mapping from docker CLI to crictl
```
We usually capitalize abbreviations.
It's OK to skip that as this is moving existing content.

The exact versions for below mapping table are for docker cli v1.40 and crictl v1.19.0. Please note that the list is not exhaustive. For example, it doesn't include experimental commands of docker cli.

{{< note >}}

> The output format of CRICTL is similar to Docker CLI, despite some missing columns for some CLI. Make sure to check output for the specific command if your script output parsing.

Reviewer 2: ```suggestion
The output format of CRICTL is similar to Docker CLI, despite some missing columns for some CLI. Make sure to check output for the specific command if your command output is being parsed programmatically.
```

{{< /note >}}

> ### Retrieve Debugging Information

Reviewer: nit:
```suggestion
### Retrieve debugging information
```

It's OK to skip that as this is moving existing content.
